tests/e2e/gateway/00-install.yaml

```diff
@@ -0,0 +1,157 @@
+---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: dex-config
+data:
+  dex.yaml: |-
+    issuer: http://dex.svc:30556/dex
+    storage:
+      type: sqlite3
+      config:
+        file: /tmp/dex.db
+    web:
+      http: 0.0.0.0:5556
+    telemetry:
+      http: 0.0.0.0:5558
+    logger:
+      level: "debug"
+    oauth2:
+      passwordConnector: local
+    staticClients:
+      - id: test
+        name: test
+        secret: ZXhhbXBsZS1hcHAtc2VjcmV0
+        redirectURIs:
+          - http://tempo-foo-gateway.svc:8080/oidc/test-oidc/callback
+    enablePasswordDB: true
+    staticPasswords:
+      - email: "admin@example.com"
+        # bcrypt hash of the string "password"
+        hash: "$2a$10$2b2cU8CPhOTaGrs1HRQuAueS7JTT5ZHsHSzYiFPm1leZck7Mc8T4W"
+        username: "admin"
+        userID: "08a8684b-db88-4b73-90a9-3cd1661f5466"
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  labels:
+    app: dex
+  name: dex
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: dex
+  strategy:
+    type: RollingUpdate
+  template:
+    metadata:
+      labels:
+        app: dex
+    spec:
+      containers:
+        - args:
+            - dex
+            - serve
+            - /data/dex/dex.yaml
+          image: dexidp/dex:v2.30.0
+          imagePullPolicy: IfNotPresent
+          name: hydra
+          env:
+          ports:
+            - containerPort: 5556
+              name: public
+          resources: {}
+          volumeMounts:
+            - mountPath: /data/dex/
+              name: dex-config
+              readOnly: true
+      volumes:
+        - configMap:
+            name: dex-config
+          name: dex-config
+---
+apiVersion: v1
+kind: Service
+metadata:
+  name: dex
+spec:
+  ports:
+    - name: public
+      port: 30556
+      protocol: TCP
+      targetPort: public
+  selector:
+    app: dex
+---
+apiVersion: v1
+kind: Secret
+metadata:
+   name: minio-test
+stringData:
+  endpoint: http://minio.minio.svc:9000
+  bucket: tempo
+  access_key_id: tempo
+  access_key_secret: supersecret
+type: Opaque
+---
+apiVersion: v1
+kind: Secret
+metadata:
+   name: oidc-test
+stringData:
+  clientID: test
+  clientSecret: ZXhhbXBsZS1hcHAtc2VjcmV0
+type: Opaque
+---
+apiVersion: tempo.grafana.com/v1alpha1
+kind: TempoStack
+metadata:
+  name: foo
+spec:
+  images:
```

Comment:
Do we need to add the images? It will force us to modify this file for each release, right?

Reply:
It doesnt work with the 2.0.1 images. Next time we update the file, we can remove it.